packages/cli/tests/unit/inspect.spec.ts

```diff
@@ -11,9 +11,9 @@ import { CodeObject, CodeObjectMatch } from '../../src/search/types';
 tmp.setGracefulCleanup();
 
 const fixtureDir = join(__dirname, 'fixtures', 'ruby');
-const appMapDir = tmp.dirSync().name;
+const appMapDir = tmp.dirSync().name.split('\\').join('/');
 
-const now = Date.now();
+const now = Date.now() / 1000;
```

Comment:
This again..?

Reply:
[Apparently, `utimesSync` takes timestamps in seconds, but `Date.now()` generates them in milliseconds](https://github.com/nodejs/node/issues/33227#issuecomment-623391936). But I'm confused as to why it works in Linux / Mac if that's the case...